src/test/unit_tests/test_ns_list_objects.js

```diff
@@ -67,7 +67,8 @@ function test_ns_list_objects(ns, object_sdk, bucket) {
             const r = await ns.list_objects({
                 bucket,
                 delimiter: '/',
-                prefix: 'folder'
+                prefix: 'folder',
+                path: dir_path
```

Comment:
Why is `path` now in the params to all of the ns operations?
I didn't see it coming...

Reply:
it's the directory path and it's not saved on the namespace resource, there are other solutions I thought about, we can do a discussion which is preferred.

Reply:
according to our meeting, can I resolve this comment?